cloud-functions/auth-redirect/templates/index.html

```diff
@@ -1,9 +1,17 @@
 <html>
 <head>
 	<title>Success code={{ authorization_code }}</title>
+	<link rel="stylesheet" type="text/css" href="index.css">
```

Comment:
change this import to:
```jinja
<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='styles.css') }}">
```

this page is being rendered by jinja and won't load index.css from the `templates` folder. the above syntax will make it work (assuming you moved it to `cloud-functions/auth-redirect/static/styles.css`